uni/lib/view/restaurant/cantine_page_view.dart

```diff
@@ -0,0 +1,176 @@
+  void initState() {
+    super.initState();
+    final int weekDay = DateTime.now().weekday;
+    super.initState();
+    tabController = TabController(vsync: this, length: daysOfTheWeek.length);
+    final offset = (weekDay > 5) ? 0 : (weekDay - 1) % daysOfTheWeek.length;
+    tabController.animateTo((tabController.index + offset));
+    scrollViewController = ScrollController();
+  }
+
+  @override
+  Widget getBody(BuildContext context) {
+    return StoreConnector<AppState, Tuple2<List<Restaurant>, RequestStatus?>>(
+        converter: (store) {
+          return Tuple2(store.state.content['restaurants'],
+              store.state.content['restaurantsStatus']);
+        },
+        builder: (context, restaurantsInfo) =>
+            _getPageView(restaurantsInfo.item1, restaurantsInfo.item2));
+
+  }
+
+  Widget _getPageView(List<Restaurant> restaurants, RequestStatus? status) {
+    if (status == null) {
+      return const Text("Estado do request inválido");
+    }
+    return Column(children: [
+      ListView(scrollDirection: Axis.vertical, shrinkWrap: true, children: [
+        Container(
+          padding: const EdgeInsets.fromLTRB(20, 20, 20, 10),
+          alignment: Alignment.center,
+          child: const PageTitle(name: 'Ementas', center: false, pad: false),
+        ),
+        TabBar(
+          controller: tabController,
+          isScrollable: true,
+          tabs: createTabs(context),
+        ),
+      ]),
+      RequestDependentWidgetBuilder(
+          context: context,
+          status: status,
+          contentGenerator: createTabViewBuilder,
+          content: restaurants,
+          contentChecker: restaurants.isNotEmpty,
+          onNullContent: const Center(child: Text('Não há refeições disponíveis.')))
+    ]);
+  }
+
+  Widget createTabViewBuilder(dynamic restaurants, BuildContext context) {
+      final List<Widget> dayContents =  daysOfTheWeek.map((dayOfWeek) {
+        List<Widget> cantinesWidgets = [];
+        if (restaurants is List<Restaurant>) {
+          cantinesWidgets = restaurants
+              .map((restaurant) => createCantine(context, restaurant, dayOfWeek))
+              .toList();
+        }
+        return ListView( children: cantinesWidgets,);
+      }).toList();
+
+      return Expanded(
+          child: TabBarView(
+        controller: tabController,
+        children: dayContents,
+      ));
+  }
+
+  List<Widget> createTabs(BuildContext context) {
+    final List<Widget> tabs = <Widget>[];
+
+    for (var i = 0; i < daysOfTheWeek.length; i++) {
+      tabs.add(Container(
+        color: Theme.of(context).backgroundColor,
+        child: Tab(key: Key('cantine-page-tab-$i'), text: toString(daysOfTheWeek[i])),
+      ));
+    }
+
+    return tabs;
+  }
+
+  Widget createCantine(context, Restaurant restaurant, DayOfWeek dayOfWeek) {
+      return Column(
+          mainAxisSize: MainAxisSize.min,
+          children: [
+        const SizedBox(height: 18,),
+        Card(
```

Comment:
I'm not a fan of implementing a custom card for this page only. Can you try using a generic card (same as the homepage), with the restaurant name as the title and the meals description as the child? That would get rid of the divider and the custom padding, which makes sense to me for consistency.